src/components/experiences/feedback-experiences.js

```diff
@@ -90,10 +90,10 @@ export default function FeedbackExperiences(props) {
 
 const Content = styled(DialogContent)`
   overflow-y: visible;
-  padding: 0 24px;
+  padding: ${window.innerWidth < 960 ? '0px' : '0 24px'};
```

Comment:
wrap ternaries in ()

Reply:
ternaries are now wrapped in parenthesis